src/main/kotlin/tanvd/grazi/ide/ui/SwingKT.kt

```diff
@@ -1,11 +1,12 @@
 package tanvd.grazi.ide.ui
 
+import com.intellij.ui.IdeBorderFactory
 import com.intellij.ui.components.JBTabbedPane
 import org.jetbrains.annotations.PropertyKey
 import tanvd.grazi.GraziBundle
 import java.awt.*
 import javax.swing.*
-import javax.swing.border.CompoundBorder
+import javax.swing.border.Border
 
 fun panel(layout: LayoutManager = BorderLayout(0, 0), body: JPanel.() -> Unit) = JPanel(layout).apply(body)
```

Comment:
Verify, that we should not use `JBPanel` or something like it

Reply:
JBPanel is just JPanel with some additional methods, which we don't need